.github/workflows/gradle.yml

```diff
@@ -62,3 +62,24 @@ jobs:
       with:
         name: reports-java-11
         path: '*/build/reports'
+    - name: Check current state of test262.properties
+      id: hashBefore
+      run: echo "hash=${{ hashFiles('./tests/testsrc/test262.properties') }}" >> $GITHUB_OUTPUT
+    - name: Run test262 tests
+      env:
+        RHINO_TEST_JAVA_VERSION: "11"
+      run: >-
+        ./gradlew :tests:test --tests Test262SuiteTest -DupdateTest262properties
+    - name: Check updated state of test262.properties
+      id: hashAfter
+      run: echo "hash=${{ hashFiles('./tests/testsrc/test262.properties') }}" >> $GITHUB_OUTPUT
+    - name: Compare test262 results
+      run: |
+        if [ "${{ steps.hashBefore.outputs.hash }}" != "${{ steps.hashAfter.outputs.hash }}" ]; then
```

Comment:
You might check for modification with `git diff` / `git status` here.

How long does this step take?
As long as I understand, this step executes ALL 262 tests... And the java 11 test run executes then again (but only the not excluded ones) - so it should be possible to exclude them (or regenerate the props file in the java 11 test run) 

I'm fine with your changes and the above are suggestions, that can (or not) be done in further PRs 👍